src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -212,12 +212,13 @@ struct aie_core_info_sysfs
     boost::property_tree::ptree ptarray;
     aie_metadata_info aie_meta = get_aie_metadata_info(device);
     const std::string aiepart = std::to_string(aie_meta.shim_row) + "_" + std::to_string(aie_meta.num_cols);
+    const aie_sys_parser asp(aiepart);
 
     /* Loop each all aie core tiles and collect core, dma, events, errors, locks status. */
-    for (int i = 0; i < aie_meta.num_cols; ++i)
+    for (int i = 0; i < aie_meta .num_cols; ++i)
```

Comment:
Did you mean to have the space before the '.'?

Reply:
No, I've pushed a commit to remove that unintentional space.